frontend/packages/schema-editor/src/components/SchemaInspector/ItemFieldsTab/ItemFieldsTable/ItemFieldsTableRow.tsx

```diff
@@ -8,8 +8,6 @@ import { Button, NativeSelect, Switch } from '@digdir/design-system-react';
 import { AltinnConfirmDialog } from 'app-shared/components';
 import { setRequired, setPropertyName } from '@altinn/schema-model';
 import { useTranslation } from 'react-i18next';
-import { useDispatch } from 'react-redux';
-import { removeSelection } from '../../../../features/editor/schemaEditorSlice';
 import { TrashIcon } from '@navikt/aksel-icons';
```

Comment:
I know this is not changes in this PR, but maybe we should update the import to use ```@studio/icons```?